keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java

```diff
@@ -145,4 +151,60 @@ public Response executeActionsEmail(@QueryParam("lifespan") Integer lifespan, Li
             return ErrorResponse.error("Failed to send execute actions email", Status.INTERNAL_SERVER_ERROR);
         }
     }
+
+    public static class UserWithOverridenEmail extends UserModelDelegate {
+        private String email;
+
+        public UserWithOverridenEmail(UserModel user, String email) {
+            super(user);
+            this.email = email;
+        }
+
+        @Override
+        public String getEmail() {
+            return StringUtils.defaultIfBlank(email, super.getEmail());
+        }
+    }
+
+    @Path("send-email")
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response sendMail(@QueryParam("subject") String subjectFormatKey, @QueryParam("template") String template, @QueryParam("recipient") String recipient, Map<String, String> parameters) {
+        if (StringUtils.isBlank(template)) {
+            return ErrorResponse.error("Template email missing", Status.BAD_REQUEST);
+        }
+        if (StringUtils.isBlank(subjectFormatKey)) {
+            return ErrorResponse.error("Subject missing", Status.BAD_REQUEST);
+        }
+        UserWithOverridenEmail userWithEmail = new UserWithOverridenEmail(user, recipient);
+        if (StringUtils.isBlank(userWithEmail.getEmail())) {
+            return ErrorResponse.error("User email missing", Status.BAD_REQUEST);
+        }
+
+        if (!user.isEnabled()) {
```

Comment:
It might be useful to be able to send an email to a user for example to inform its account has been disabled. Do we really want to prevent such scenario ?

Reply:
As seen together, we keep it like this and will advise later.